ScoutSuite/providers/aws/rules/findings/organizations-opt-out-policies-not-on-all-accounts.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "Opt-out policies not on all accounts",
+  "rationale": "Opt-out policies should be turned on in all accounts",
```

Comment:
You will likely need to point to some reference or link in AWS's documentation about why this is a recommended course of action.